homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,291 @@
+        """Get the charging_time."""
+        return int(data.charging_time)
+
+
+@dataclass
+class CurrentSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Current sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> float:
+        """Get the current."""
+        return float(data.current)
+
+
+@dataclass
+class InstantPowerSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Instant Power sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> float:
+        """Get the instant_power."""
+        return float(data.instant_power)
+
+
+@dataclass
+class SessionEnergySensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Session Energy sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> float:
+        """Get the session_energy."""
+        return float(data.session_energy)
+
+
+@dataclass
+class TemperatureSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Temperature sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> float:
+        """Get the temperature."""
+        return float(data.temperature)
+
+
+@dataclass
+class TotalChargedEnergySensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Total Charged Energy sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> int:
+        """Get the total_charged_energy."""
+        return int(data.total_charged_energy)
+
+
+@dataclass
+class VoltageSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Voltage sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> float:
+        """Get the voltage."""
+        return float(data.voltage)
+
+
+@dataclass
+class InstallCurrentSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Install Current sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> int:
+        """Get the install_current."""
+        return int(data.install_current)
+
+
+@dataclass
+class DynamicCurrentSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Dynamic Current sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> int:
+        """Get the dynamic_current."""
+        return int(data.dynamic_current)
+
+
+@dataclass
+class LedMaxBrightnessSensorEntityDescription(LektricoSensorEntityDescription):
+    """A class that describes the Lektrico Led Max Brightness sensor entity."""
+
+    @classmethod
+    def get_native_value(cls, data: Any) -> int:
+        """Get the led_max_brightness."""
+        return int(data.led_max_brightness)
```

Comment:
Same as with the binary sensor, once the entity description accepts a lambda, these are all no longer needed, and the lambda can just be passed in the entity description definitions below.

Reply:
I made the changes, thanks.